src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -39,20 +34,26 @@ const AddInstructors = () => {
     }
   };
 
-  const handleInstructorInput = (e) => {
```

Comment:
@Squirrel18 @anfbermudezme I changed the components form  controlled to uncontrolled so when submit the form I will get the values and don't have the problem with the select